internal/csi-addons/rbd/network_fence.go

```diff
@@ -178,5 +188,78 @@ func (fcs *FenceControllerServer) GetFenceClients(
 		},
 	}
 
+	if options[disableAutoUnfenceKey] != "false" {
```

Comment:
IMO we dont need it we already have separate API to fence and unfence, lets keep it as it is